input/pagecontent/iti-78.md

```diff
@@ -25,13 +25,22 @@ A mobile app wants to query patients, which participate in the Swiss EPR.
 The message semantics is the same as defined in [2:3.78.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378412-message-semantics) with a restriction if there are more than 5 matches. If there are more than 5 matches, the result should return zero matches.
```

Comment:
```suggestion
The message semantics is the same as defined in [2:3.78.4.1.2](https://profiles.ihe.net/ITI/PDQm/ITI-78.html#2378412-message-semantics) with a restriction if there are more than 5 matches. If there are more than 5 matches, the result should return zero matches with an OperationOutcome requesting more query parameters.
```

Without an OperationOutcome, an empty search result should be considered successful and the client may not understand that a query refinement is required.


https://www.hl7.org/fhir/search.html#errors:

> When the search fails, a server SHOULD return an [OperationOutcome](https://www.hl7.org/fhir/operationoutcome.html) detailing the cause of the failure. Note that an empty search result is not an error.